### DISCUSSION TITLE
Make the key used for absolute scroll in song select a bind in the inputs for song select

### DISCUSSION THREAD

**Discussion by Rick-Bossen** [ppy/osu#26903] [Ideas] [answered]
This is something that has been bothering me while making the switch from opentabletdriver to the built in tablet driver in laser. When using opentabletdriver, I have one of the pen buttons on my tablet bound to m2 to allow me to use absolute scroll without picking up my mouse. This is not possible in laser AFAIK, so my proposal for this would be to implement the ability to rebind the key used for absolute scroll in song select.

**Comment by Rekunan** [comment]
Adding on, it would also be good to let us rebind the pen buttons within the ingame tablet settings.

**Comment by Glinte** [comment]
This is one of the largest pain points I have on lazer.

**Reply by peppy** [reply, marked as answer]
I'll consider a solution for this whilst we redo song select.

**Reply by ram398** [reply]
yes , you are right.

**Comment by Rick-Bossen** [comment]
Seems like this will be fixed in the next release so I'll close this discussion: https://github.com/ppy/osu/pull/31604
